datadog_lambda/tracing.py

```diff
@@ -689,10 +874,20 @@ def create_inferred_span_from_http_api_event(event, context):
         "span_type": "http",
     }
     tracer.set_tags({"_dd.origin": "lambda"})
+    inferred_span_start_ns = None
+    if decode_authorizer_context:
+        injected_authorizer_data = get_injected_authorizer_data(
+            event, _EventSource(EventTypes.API_GATEWAY, EventSubtypes.HTTP_API)
+        )
+        if injected_authorizer_data:
+            inferred_span_start_ns = int(
+                injected_authorizer_data.get(Headers.Parent_Span_Finish_Time)
+            )
+    inferred_span_start_ns = inferred_span_start_ns or request_time_epoch_ms * 1e6
     span = tracer.trace("aws.httpapi", **args)
     if span:
         span.set_tags(tags)
-    span.start = request_time_epoch / 1000
```

Comment:
Is moving this into the conditional also a bug fix? Along with fixing the request time epoch?

Reply:
Yes, assuming the conditional makes sense. However, after taking a closer look, `tracer.trace()` should always return a span. So I'll actually remove this `if span:` line.